doc/osh.1

```diff
@@ -5,45 +5,47 @@
 .Nm osh
 .Nd A bash-compatible UNIX shell
 .Sh SYNOPSIS
-.Nm 
+.Nm
 .Op Ar options
 .Ar script
 .Op Ar args
-.Br
-.Nm 
+.Nm
 .Op Ar options
 .Fl c
 .Op command_string
 .Op args
 .Sh DESCRIPTION
-OSH is a command-line shell. It behaves like busybox and `osh` is a symlink to it.
+.Nm OSH
```

Comment:
This should instead start like so:

```mdoc
The
.Nm
utility is a command-line shell.
```

`Nm` automatically inserts the name of the program, no need to put it here.